### ISSUE TITLE
cannot access SpotifyApi.withAccessToken in version spotify: ^0.6.0

### ISSUE THREAD

**Issue by danielconvrtx** [rinukkusu/spotify-dart#113]

**Comment by stulov** [comment]
Confirm this. The change is currently only on master branch and hasn't been released yet. I personally temporarily added the package as a git dependency and that solved the issue

**Comment by fotiDim** [comment]
Only @rinukkusu AFAIK can make releases so he has to let us know if he can do it.
For now you can follow @stulov 's suggestion. Add this to your `pubspec.yaml`:

```
spotify:
    git: https://github.com/rinukkusu/spotify-dart
```

**Comment by rinukkusu** [comment]
Hi, major oversight - whoopsie, it's released! - `0.7.0` is on pub.dev
